File: src/main/kotlin/net/wavem/uvc/rms/domain/job/JobInfo.kt
package net.wavem.uvc.rms.domain.job

import net.wavem.uvc.rms.domain.job.result.JobResult
import net.wavem.uvc.rms.domain.job.task.TaskInfo

data class JobInfo(
    val jobPlanId: String,
    val jobGroupId: Int,
    val jobOrderId: Int,
    val taskInfo: TaskInfo?,
    val jobResult: JobResult?
) {
}

File: src/rms/response/location/domain/jobInfo/job_info.py

class JobInfo():
    def __init__(
        self,
        jobPlanId: str = None,
        jobGroupId: str = None,
        jobOrderId: str = None,
        taskInfo: dict = None
    ) -> None:
        self.jobPlanId: str = jobPlanId
        self.jobGroupId: str = jobGroupId
        self.jobOrderId: str = jobOrderId
        self.taskInfo: dict = taskInfo
        
        
__all__ = ['job_info']